apps/NES/elpi/nes.elpi

```diff
@@ -152,6 +152,27 @@ current-path NS :- std.do! [
   std.rev Stack NS,
 ].
 
+% Find an existing namespace, or panic.
```

Comment:
* We could use some examples. The use case here is `NES.Begin A. NES.Begin B. (* ... *) NES.End B. NES.Open B. (* rather than A.B *)`.
* The choice of `.Path` for "use the top-level `Path`" interacts poorly with the syntax of Elpi commands (e.g., one must write `NES.List ".Path".`).